python/tvm/relay/frontend/pytorch_utils.py

```diff
@@ -25,3 +35,98 @@ def is_version_greater_than(ver):
     return "".join(re.findall(r"(\d+\.)(\d+\.)(\d)", torch.__version__)[0]) > "".join(
         re.findall(r"(\d+\.)(\d+\.)(\d)", ver)[0]
     )
+
+
+def batched_nms_pattern(boxes, scores, idxs, iou_threshold):
+    """A pattern to detect batched_nms function in torchvision"""
```

Comment:
Can we have more comments about this pattern matching processing? I'm a bit confused of how class id is restored.

Reply:
Ok I'll update after the CI finishes. In the mean time, you can have a look at rewrite test cases in https://github.com/apache/tvm/blob/main/tests/python/relay/test_dataflow_pattern.py#L698-L736 to get an idea of how they work

Reply:
I think I understand what the rewritten pattern does, but I'm not understanding how the pytorch version works.

Reply:
I added some comments to explain the pattern. It should be clear now.